### ISSUE TITLE
fonts

### ISSUE THREAD

**Issue by moskhawas** [withspectrum/spectrum#5040]
Bug Report
I do not know if this is intended, but input, button and textarea elements have different font from the rest of the app as they do not normally inherit the fonts from the body or html. 
I could add a pull request to fix it if you want.

**Comment by brianlovin** [comment]
That would be super helpful @moskhawas thank you!

**Comment by lachlanjc** [comment]
YES YES YES it's so small but I notice it every time I use Spectrum & it bugs me :)
